tutorindigo/templates/indigo/lms/static/sass/courseware/_discover.scss

```diff
@@ -136,6 +138,109 @@
         clear: both;
         content: "";
     }
+    .content-area {
+        display: flex; // Align the sidebar and courses horizontally
+        flex-direction: row;
+        gap: 16px; // Add spacing between the sidebar and courses
+        align-items: flex-start;
+
+        .course-holder {
+            flex-grow: 1; // Allow the courses to take up the remaining space
+            width: 100%; // Ensure it spans the available space
+        }
+    }
+    .filter-sidebar {
+        flex: 0 0 25%; // Set the sidebar to always occupy 25% of the width
+        max-width: 25%;
+        background: $primary-light;
+        padding: 15px;
+        border-radius: 6px;
+        box-shadow: 0px 0px 0px 1px rgba(0, 0, 0, 0.05),
+                    0px 4px 6px -2px rgba(0, 0, 0, 0.05),
+                    0px 10px 15px -3px rgba(0, 0, 0, 0.10);
+        flex-shrink: 0;
+        height: auto;
+    }
+
+    .filter-sidebar .search-facets {
+        width: 100%;
+        margin: 0;
+        box-shadow: none;
+        background: none;
+        border-top: none;
+        border-bottom: none;
+    }
+
+    .filter-sidebar .search-facets-lists {
+        padding: 10px 0;
+    }
+
+    .filter-sidebar .search-facets-lists h3 {
+        border: none;
+        padding: 0;
+        margin: 0 0 10px;
+        color: $primary;
+        font-size: 14px;
+        font-weight: 700;
+    }
+
+    // Styles for filter options with checkbox-like behavior
+    .search-facets-lists {
+        ul.facet-list {
+            padding: 0 0 0 14px;
+            max-height: 100px;
+            overflow-y: auto;
+        }
+        
+        ul.facet-list li {
+            display: flex;
+            align-items: center;
+            margin-bottom: 10px;
+        }
+
+        button.facet-option {
+            flex-grow: 1;
+            position: relative;
+            float: none;
+            width: auto;
+            background: none !important;
+            padding: 0; /* Space for checkbox */
+            font-size: 14px;
+            line-height: 20px;
+            color: #191B23;
+            cursor: pointer;
+            border: none;
+            display: flex;
+            align-items: center;
+            gap: 10px;
+        }
+        button.facet-option:hover {
+            color: #191B23;
+        }
+
+        button.facet-option::before {
+            content: "";
+            width: 16px;
+            height: 16px;
+            border: 2px solid #191B23;
+            border-radius: 4px;
+            margin-right: 8px;
+            display: inline-block;
+        }
+
+        button.facet-option.selected::before {
+            content: "\f00c";
+            font-family: "Font Awesome 5 Free";
+            font-weight: 900;
+            background: $primary;
+            text-align: center;
+            line-height: 16px;
+            display: flex;
+            align-items: center;
+            justify-content: center;
+        }
+        
+    }
```

Comment:
```suggestion
        button.facet-option.selected::before {
        content: "\f00c";
        font-family: "Font Awesome 5 Free";
        font-weight: 900;
        background: #0067B1;
         text-align: center;
         line-height: 16px;
         top: 0;
        color: #FFF;
        border: 2px solid #0067B1;
        }
        
    }
```

I couldn't maintain the original indentation, sorry :( 
![image](https://github.com/user-attachments/assets/1333a264-268f-43b3-82a5-68c16f163756)

Reply:
Don't worry. Thanks ✨